packages/preset-icons/src/index.ts

```diff
@@ -95,6 +96,11 @@ export const preset = (options: IconsOptions = {}): Preset => {
           return
         }
 
+        if (unit) {
```

Comment:
this should be done with custom `iconCustomizer` like on `unplugin-icons` and only if the w/h on props are missing:
- the logic to apply will be something similar to this https://github.com/iconify/iconify/blob/master/packages/utils/src/loader/modern.ts#L42 only for width and height props
- you need to provide a custom `iconCustomizer` like this (with previous logic after provided `iconCustomizer`): https://github.com/antfu/unplugin-icons/blob/main/src/core/loader.ts#L78

Reply:
we also need to provide a test to check the w/h aren't overridden

Reply:
@antfu maybe it will be better send it upstream